OpenSim/Common/ComponentConnector.h

```diff
@@ -446,17 +491,9 @@ class OSIMCOMMON_API AbstractInput : public AbstractConnector {
             channelName = "";
         }
         
-        // Annotation.
-        if (leftParen != std::string::npos && rightParen != std::string::npos) {
-            annotation = connecteeName.substr(leftParen + 1,
-                                              rightParen - (leftParen + 1));
-        } else {
-            if (!channelName.empty()) {
-                annotation = channelName;
-            } else {
-                annotation = outputName;
-            }
-        }
+        // Alias.
+        alias = connecteeName.substr(leftParen + 1, rightParen - (leftParen + 1));
```

Comment:
I now see that....I was having difficulty navigating the PR. Thanks for clarifying